imports/plugins/included/product-variant/client/templates/products/productGrid/controls.html

```diff
@@ -13,8 +13,8 @@
 
       {{#if product.isDeleted}}
         <div>
-        <span class="badge badge-danger" data-i18n="app.deleted">
-          <span>Deleted</span>
+        <span class="badge badge-danger" data-i18n="app.archived">
+          <span>Archived</span>
```

Comment:
This works. The inner span is because Blaze and jQuery fight each other for who has the right to update the text contents of an element. The `<span>` around the `Archived` prevents this from happening. Don't know why this is... but this solution seems to fix that issue.